ui-tests/utils/constants.js

```diff
@@ -7,6 +7,7 @@ const routes = {
   public: {
     home: appBaseUrl,
     literatureSearch: `${appBaseUrl}/literature`,
+    literatureDetail: `${appBaseUrl}/literature/1`,
```

Comment:
this should be a function `getLiteratureDetailForId` or `literatureDetailForRecord1` so that from reading the tests we can understand what's being tested more clearly.